sonar-db/src/main/java/org/sonar/db/permission/GroupPermissionMapper.java

```diff
@@ -59,4 +59,6 @@
   List<String> selectAnyonePermissions(@Nullable @Param("projectId") Long projectId);
 
   void insert(GroupPermissionDto dto);
+
+  void deleteByProjectId(@Param("projectId") long projectId);
```

Comment:
I think should rather be `deleteByProjectId` as there is really nothing enforcing the argument is the id of a project. So, it can actually be any component, such as view or developer (or even file even though it can really not exist)